corgi/core/migrations/0081_fix_build_sources.py

```diff
@@ -0,0 +1,60 @@
+# Generated by Django 3.2.18 on 2023-06-19 02:18
+
+import os
+
+from django.db import migrations
+from django.db.models import Value, functions
+
+
+def fix_build_source_urls(apps, schema_editor):
```

Comment:
perhaps I am missing something - I was expecting an invoke clean_source_of_build() at least as a final check here or is the data migration itself require a different code path ?

Reply:
thx for explanation, LGTM!